tests/core/test_flare.py

```diff
@@ -233,12 +238,20 @@ def test_uri_no_verify_ssl(self, mock_config, mock_tempdir, mock_strftime):
         f.set_ssl_validation(request_options)
         self.assertFalse(request_options['verify'])
 
-    @attr(requires='core_integration')
```

Comment:
I only left the `test_endpoint` test in `core_integration` because I thought that all the other ones were more unit tests or mock tests than actual integration tests. No other reason though.